MalApi/MyAnimeListApi.cs

```diff
@@ -72,6 +78,40 @@ private HttpRequestMessage InitNewRequest(string uri, HttpMethod method)
             return request;
         }
 
+        private HttpRequestMessage InitNewRequestWithCredentials(string user, string password, string uri, HttpMethod method, UpdateObjectBase updateInfo = null)
```

Comment:
Consider moving uri and method to be the first two parameters, as they are kind of the "primary" parameters of the method.

Reply:
updateInfo does not belong in InitNewRequestWithCredentials. When functionality for adding and removing of anime/manga entries is added, that will use credentials but will not take the same input that updating does. Any MAL endpoint-specific logic can be done in the method for that endpoint on top of the HttpRequestMessage object returned by InitNewRequest or InitNewRequestWithCredentials.

Reply:
Technically, delete takes only an id as input while add and update take the same input as far as I understand from the API. But again I agree that setting the content inside InitNewRequestWithCredentials is not the best solution due to possible future changes to the API and the functionality that the method name reflects.